lib/rucio/core/did.py

```diff
@@ -1606,7 +1606,7 @@ def list_dids(scope, filters, did_type='collection', ignore_case=False, limit=No
 
 @read_session
 def list_dids_extended(scope, filters, did_type='collection', ignore_case=False, limit=None,
-                       offset=None, long=False, recursive=False, session=None):
+                       offset=None, long=False, recursive=False, ignore_dids=None, session=None):
```

Comment:
A test would be nice, have a look at https://github.com/rucio/rucio/blob/c857f58149b09d785338c0abe02e3487b85f4cd9/lib/rucio/tests/test_did_meta_plugins.py

Reply:
added, but not here (to filter engine)